client/components/forms/docs/example.jsx

```diff
@@ -89,8 +98,7 @@ var FormFields = React.createClass( {
 							id="username"
 							name="username"
 							disabled
-							placeholder="Placeholder text..."
-						/>
+							placeholder="Placeholder text..." />
```

Comment:
It looks like this and the following changes may have been introduced by an auto formatter? Could we remove them from the PR? Whenever we have formatting issues we like to separate them into their own PRs to make it easy to pinpoint changes with revisions. In this case, it's hard to tell if something changed because of actual logic changes or because we missed something in the formatting update.